src/button/button-bar.jsx

```diff
@@ -1,20 +1,9 @@
 import React from 'react';
-import styled from 'styled-components';
 import Button from './button';
 
-const Main = styled.div`
-  display: flex;
-  flex-direction: column;
-  z-index: 99;
-`;
-const FirstRow = styled.div`
-  color: ${({ theme }) => `${theme.inverted}`};
-  display: flex;
-`;
-const SecondRow = styled.div`
-  color: ${({ theme }) => `${theme.inverted}`};
-  display: flex;
-`;
+import Main from './main.styled';
```

Comment:
Using this as an example but for any component using multiple styled components could you make __one__ `.styled.js` file and name export the multiple styled components

So in this case
`button-bar.styled.js`

```javascript
export const Main=styled.div` ... `
export const FirstRow = styled.div ` ..`
export const SecondRow = styled.div ` ..`
```

`button-bar.jsx````javascript
import{ Main, FirstRow, SecondRow } from './button-bar.styled
```

Once thats done this is awesome :)

Reply:
Aight, I'm on it.